build/apis/exchange-chain-x-chain-api.md

```diff
@@ -575,10 +575,74 @@ curl -X POST --data '{
 }
 ```
 
+### avm.export
+
+Send a non-AVAX from the X-Chain to the C-Chain.  
+After calling this method, you must call [`avax.import`](https://docs.avax-dev.network/build/apis/contract-chain-c-chain-api#avax-import) on the C-Chain to complete the transfer.
+
+#### **Signature**
+
+```text
+avm.export({
+    to: string,
+    amount: int,
+    assetID: string,
+    from: []string, (optional)
+    changeAddr: string, (optional)
+    username: string,
+    password:string,
+}) ->
+{
+    txID: string,
+    changeAddr: string,
+}
+```
+
+* `to` is the P-Chain address the asset is sent to.
+* `amount` is the amount of nASSET to send.
```

Comment:
Nit:

"`amount` is the amount of nASSET to send." --> "`amount` is the amount of the asset to send."